python/ray/_raylet.pyx

```diff
@@ -165,8 +165,8 @@ from ray.includes.libcoreworker cimport (
 )
 from ray.includes.stream_redirection cimport (
     CStreamRedirectionOptions,
-    RedirectStdout,
-    RedirectStderr,
+    RedirectStdoutForOnce,
+    RedirectStderrForOnce,
```

Comment:
what does `ForOnce` mean and why is it changed now?

Reply:
`ForOnce` means "the function should be called for at most once in the process".

Reply:
> why is it changed now

The `internal` namespace has a similarly-named function `RedirectStderr`, I think it's better to suffix it as `ForOnce` for better clarity.

Reply:
got it; totally fine to change it I just didn't understand the meaning behind the name

given it isn't used frequently so concision it less important, I'd suggest naming it something like:

- `RedirectStdoutOncePerProcess`
- `RedirectStdoutGlobally`

Reply:
and just so it's clear, the reason I'm suggesting the change is because `ForOnce` is a bit misleading. It actually made me think that this was a temporary redirection that was only scoped to the duration of a task/actor method or similar

Reply:
updated to `RedirectStdoutOncePerProcess`